src/main/java/com/synopsys/integration/alert/configuration/EventListenerConfigurer.java

```diff
@@ -1,7 +1,7 @@
 /*
  * blackduck-alert
  *
- * Copyright (c) 2021 Synopsys, Inc.
+ * Copyright (c) 2022 Synopsys, Inc.
```

Comment:
You will probably need to merge master into this branch, these files should already have 2022 (According to github master)